PhysicsTools/NanoAOD/python/met_cff.py

```diff
@@ -21,6 +21,8 @@
        covXY = Var("getSignificanceMatrix().At(0,1)",float,doc="xy element of met covariance matrix", precision=8),
        covYY = Var("getSignificanceMatrix().At(1,1)",float,doc="yy element of met covariance matrix", precision=8),
        significance = Var("metSignificance()", float, doc="MET significance",precision=10),
+       metUnclustEnUp = Var("shiftedPt('UnclusteredEnUp')", float, doc="MET Unclustered Energy Up ",precision=10),
+       metUnclustEnDown = Var("shiftedPt('UnclusteredEnDown')", float, doc="MET Unclustered Energy Down ",precision=10),
```

Comment:
are up and down symmetric by construction as for some other JME uncertainties or not?

Reply:
From the formulas for the correction it is hard to see how the uncertainty would be symmetric on an event-by-event basis:
https://twiki.cern.ch/twiki/bin/viewauth/CMS/METType1Type2Formulae#4_The_Type_II_correction

Reply:
from here: 
https://github.com/cms-sw/cmssw/blob/master/PhysicsTools/PatUtils/python/tools/runMETCorrectionsAndUncertainties.py#L994
and here:
https://github.com/cms-sw/cmssw/blob/09c3fce6626f70fd04223e7dacebf0b485f73f54/PhysicsTools/PatUtils/plugins/ShiftedParticleProducer.cc#L45

I'd say the uncertainty is symmetric by construction

Reply:
The reason I question the symmetry assumption is the comparison between the three MET types in TTbar MC:
![met](https://user-images.githubusercontent.com/1213276/31120743-805dbb16-a82d-11e7-8196-11b4a28fd457.png)

Reply:
it is not easy to see a symmetry here, you should do the ratio of the variations wrt nominal

Reply:
Good point, here is everything normalised to MET central value:
![met](https://user-images.githubusercontent.com/1213276/31121978-2185b40e-a832-11e7-8160-57375c17e6ff.png)

Reply:
@kreczko @arizzi You may find more studies on the validation of the uncertainties that we did in the met group here : https://indico.cern.ch/event/557439/contributions/2246809/attachments/1344891/2027219/METValidationStudies.pdf [slides 11-13]. 

One more thing: TypeII correcteded MET is not currently supported in the MET group - only Type1

Reply:
I don't see how shifting the unclustered energy up and down, even by a symmetric fraction, should in general give a symmetric shift to the total MET pt, as there's a non-linear operation in between. You probably get something symmetric either in some random lucky cases, or if you have a large met from clustered energy in the event and so you're just seeing a small (and thus linear) perturbation around it.